### DISCUSSION TITLE
Expression syntax, string interpolation inside expression scope

### DISCUSSION THREAD

**Discussion by jyeros** [community/community#25635] [Actions]
<p>Hello, I am trying to call a function and pass an string where I want to interpolate a github expression, what is the syntax to do it?</p>
<p>My specific case is I built a composite action and I want to call <code>hashFiles</code> function with path: ’<code>${{ inputs.dir }}/path/to/file'</code> something like:</p>
<pre><code class="lang-auto">${{ hashFiles('${{ inputs.dir }}/path/to/file') }}
</code></pre>
<p>The problem is that the syntax is wrong, I think I don’t need <code>${{ }}</code> to specify <code>inputs.dir</code> because I already open a expression scope, but I don’t know how to interpolate the expression inside the path.</p>

**Comment by Simran-B** [comment, marked as answer]
<p>Nested expressions are not supported, but this should do the trick:</p>
<p><code>${{ hashFiles(format('{0}/path/to/file', inputs.dir)) }}</code></p>

**Comment by jyeros** [comment]
<p>Thank you. If nested expressions are not supported, <code>format</code> looks like a good solution.</p>
